### PR TITLE
ip-route: add page

### PR DESCRIPTION
**Pull request by patricedenis**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/ip-route.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# ip route
+
+> IP Routing table management subcommand.
+> More information: <https://www.man7.org/linux/man-pages/man8/ip-route.8.html>.
```
Comment (marchersimon) [inline pages/linux/ip-route.md:4]:
```suggestion
> More information: <https://manned.org/ip-route>.
```
Maybe we should use the manned.org page here. (https://github.com/tldr-pages/tldr/issues/5143#issuecomment-812939570)

File: pages/linux/ip-route.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display the routing table:
+
+`ip route show/list`
```
Comment (marchersimon) [inline pages/linux/ip-route.md:8]:
```suggestion
`ip route show`
```
I think we should only use `show` here. Both work and do exactly the same, but `list` isn't in the man page.

File: pages/linux/ip-route.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add a default route using gateway forwarding:
+
+`sudo ip route add default via {{gateway_ip}}`
+
+- Add a default route using eth0:
+
+`sudo ip route add default dev{{eth0}}`
```
Comment (marchersimon) [inline pages/linux/ip-route.md:16]:
```suggestion
`sudo ip route add default dev {{eth0}}`
```

File: pages/linux/ip-route.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Add a static route:
+
+`sudo ip route add {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
+
+- Delete a static route:
+
+`sudo ip route del {{destination_network_ip}} dev {{eth0}}`
+
+- Change/Replace a static route:
```
Comment (marchersimon) [inline pages/linux/ip-route.md:26]:
```suggestion
- Change or replace a static route:
```

File: pages/linux/ip-route.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`sudo ip route change/replace {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
```
Comment (marchersimon) [inline pages/linux/ip-route.md:28]:
```suggestion
`sudo ip route {{change|replace}} {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
```

File: pages/linux/ip-route.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Show which route will be used by the kernel to reach one ip address:
```
Comment (marchersimon) [inline pages/linux/ip-route.md:30]:
```suggestion
- Show which route will be used by the kernel to reach a ip address:
```

**Review by marchersimon — CHANGES_REQUESTED** [review]

File: pages/linux/ip-route.md (hunk 2 above)
Comment (Waples) [reply pages/linux/ip-route.md:8]: or `ip route {{show|list}}` ?

File: pages/linux/ip-route.md (hunk 6 above)
Comment (Waples) [reply pages/linux/ip-route.md:30]: it's **an ip address**. 
ref: An Internet Protocol address (IP address)

File: pages/linux/ip-route.md (hunk 2 above)
Comment (marchersimon) [reply pages/linux/ip-route.md:8]: Is there a difference between `show` and `list`?

Comment (bl-ue) [reply pages/linux/ip-route.md:8]: If they're identical, then hmmm, I'm not sure what to do here.  Probably what @Waples suggests; that seems like the best idea.

**Comment by patricedenis** [comment]
Thank for all of you for your reviews !

**Comment by Waples** [comment]
dat `force-push` though :p haha

File: pages/linux/ip-route.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Display the routing table:
+
+`ip route {{show|list}}`
+
+- Add a default route using gateway forwarding:
+
+`sudo ip route add default via {{gateway_ip}}`
+
+- Add a default route using eth0:
```
Comment (marchersimon) [inline pages/linux/ip-route.md:14]:
```suggestion
- Add a default route using `eth0`:
```

File: pages/linux/ip-route.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+- Delete a static route:
+
+`sudo ip route del {{destination_network_ip}} dev {{eth0}}`
+
+- Change or replace a static route:
+
+`sudo ip route {{change|replace}} {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
+
+- Show which route will be used by the kernel to reach an ip address:
```
Comment (marchersimon) [inline pages/linux/ip-route.md:30]:
```suggestion
- Show which route will be used by the kernel to reach an IP address:
```

**Comment by patricedenis** [comment]
> dat `force-push` though :p haha

@Waples : Sorry I don't understrand here ^^
I'm not native english so

**Comment by Waples** [comment]
> @Waples : Sorry I don't understrand here ^^
> I'm not native english so

git force pushing can sometimes be frowned upon :p no biggy

**Comment by patricedenis** [comment]
Is that when I do a rebase to the branch, I could push without the force option?

**Comment by bl-ue** [comment]
@patricedenis under normal (which are most) conditions, you can just commit without rebasing and push without force-pushing.

In #5539 @waldyrious gave a great explanation, but it's not necessary for most circumstances.  Just `git pull` to fetch changes made on GH such as suggestion applications / others' changes, and use normal `git add ...` + `git commit ...` to commit local changes, and `git push` to push the changes up.

Even if you have a messy commit history in your PR, it doesn't matter because we squash all of the commits into one right before merging.

**Review by marchersimon — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Comment by patricedenis** [comment]
@Waples : ok I'm sorry I didn't want to be mean.
I was in fact applying what @waldyrious explained to me in #5539 as @bl-ue just mentioned.

**Comment by bl-ue** [comment]
No worries @patricedenis, when communicating via a purely textual medium it can be difficult to convey (or not convey) incorrect emotions, especially when we don't all speak the language with the same levels of ease.

File: pages/linux/ip-route.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+`sudo ip route add default via {{gateway_ip}}`
+
+- Add a default route using `eth0`:
+
+`sudo ip route add default dev {{eth0}}`
+
+- Add a static route:
+
+`sudo ip route add {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
```
Comment (navarroaxel) [inline pages/linux/ip-route.md:20]: Why not just `destination_ip`?
```suggestion
`sudo ip route add {{destination_ip}} via {{gateway_ip}} dev {{eth0}}`
```

Comment (patricedenis) [reply pages/linux/ip-route.md:20]: ok, no problemo for me. In fact, you're right, you can definitely add a route to a specific destination host.

File: pages/linux/ip-route.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+`sudo ip route add {{destination_ip}} via {{gateway_ip}} dev {{eth0}}`
+
+- Delete a static route:
+
+`sudo ip route del {{destination_network_ip}} dev {{eth0}}`
+
+- Change or replace a static route:
+
+`sudo ip route {{change|replace}} {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
```
Comment (navarroaxel) [inline pages/linux/ip-route.md:28]:
```suggestion
`sudo ip route {{change|replace}} {{destination_ip}} via {{gateway_ip}} dev {{eth0}}`
```

File: pages/linux/ip-route.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+
+`sudo ip route add default dev {{eth0}}`
+
+- Add a static route:
+
+`sudo ip route add {{destination_ip}} via {{gateway_ip}} dev {{eth0}}`
+
+- Delete a static route:
+
+`sudo ip route del {{destination_network_ip}} dev {{eth0}}`
```
Comment (navarroaxel) [inline pages/linux/ip-route.md:24]:
```suggestion
`sudo ip route del {{destination_ip}} dev {{eth0}}`
```

**Review by marchersimon — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**